docs/feature_encoders.md

```diff
@@ -158,6 +158,23 @@ bool encoder_update_user(uint8_t index, bool clockwise) {
 }
 ```
 
+Initialization code should be implemented in `encoder_init_state_kb`:
+
+```c
+void encoder_init_state_kb(uint8_t index) {
+    // set up encoder at the required index
+}
+
+To read an encoder that is not directly connected to an input pin but to another interface, implement `encoder_read_state_kb`:
+
+```c
+uint8_t encoder_read_state_kb(uint8_t index) {
```

Comment:
```suggestion
Initialization code should be implemented in `encoder_init_state`:

```c
void encoder_init_state(uint8_t index) {
    // set up encoder at the required index
}

To read an encoder that is not directly connected to an input pin but to another interface, implement `encoder_read_state`:

```c
uint8_t encoder_read_state(uint8_t index) {
```